lib/routes.js

```diff
@@ -177,6 +177,7 @@ function addSiteController(router, site, providers) {
       _.assign(site, _.pick(controller, 'resolvePublishing'));
       _.assign(site, _.pick(controller, 'resolvePublishUrl'));
       _.assign(site, _.pick(controller, 'modifyPublishedData'));
+      _.assign(site, _.pick(controller, 'assignToMetaOnPublish'));
```

Comment:
i know this is just like.. the pattern set by whoever came before, but wouldn't this be written more simply as

```js
    _.assign(site, {
      providers,
      // things to remember from controller
      _.pick(controller, 'resolvePublishing', 'resolvePublishUrl', 'modifyPublishedData', 'assignToMetaOnPublish')
    );
```

or like.. w/o lodash...

```js
    Object.assign(site, {
      providers,
      // I don't know why it's important that we not just assign everything from the controller
      resolvePublishing: controller.resolvePublishing,
      resolvePublishUrl: controller.resolvePublishUrl,
      modifyPublishedData: controller.modifyPublishedData,
      assignToMetaOnPublish: controller.assignToMetaOnPublish
    });
```